boskos/janitor/deployment.yaml

```diff
@@ -1,17 +1,43 @@
+---
+# gce-project janitor
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: boskos-janitor
+  name: boskos-gce-janitor
   labels:
-    app: boskos-janitor
+    app: boskos-gce-janitor
 spec:
   replicas: 1  # one canonical source of resources
   template:
     metadata:
       labels:
-        app: boskos-janitor
+        app: boskos-gce-janitor
     spec:
       terminationGracePeriodSeconds: 30
       containers:
-      - name: boskos-janitor
+      - name: boskos-gce-janitor
         image: gcr.io/k8s-testimages/janitor:v20170502-b59bddad
+        args:
+        - --resource-type=gce-project
+
+---
+#gke-project janitor
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: boskos-gke-janitor
+  labels:
+    app: boskos-gke-janitor
+spec:
+  replicas: 1  # one canonical source of resources
+  template:
+    metadata:
+      labels:
+        app: boskos-gke-janitor
+    spec:
+      terminationGracePeriodSeconds: 30
```

Comment:
How long does it take us to clean up a project? We should set this value at 3x so this is suggesting that it normally takes us 10s to clean a dirty project, which I suspect is faster than we can actually do